airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -79,6 +79,8 @@ def __init__(
         jinja_job_args: dict = None,
         on_kube_api_event: callable = None,
         parse_xcom_event: xcom_value_parser = xcom_value_parser,
+        labels: dict = None,
+        volume_files_from_secret: dict = None,
```

Comment:
Consider changing name: mount_files_from_secret? or secret_mounted_files? or something alike. Not sure this is clear enough.